blaze-client/src/test/scala/org/http4s/client/blaze/BlazeClientSpec.scala

```diff
@@ -145,11 +145,6 @@ class BlazeClientSpec extends Http4sSpec {
             }
 
           val allRequests = for {
-            _ <- failedRequests.handleErrorWith(_ => IO.unit)
-            _ <- failedRequests.handleErrorWith(_ => IO.unit)
-            _ <- failedRequests.handleErrorWith(_ => IO.unit)
-            _ <- failedRequests.handleErrorWith(_ => IO.unit)
-            _ <- failedRequests.handleErrorWith(_ => IO.unit)
             _ <- failedRequests.handleErrorWith(_ => IO.unit)
```

Comment:
I remember there are cases where we need to `flatmap` two failures to reproduce the error.